### DISCUSSION TITLE
How to get started with github

### DISCUSSION THREAD

**Discussion by adil-wq** [community/community#128984] [New to GitHub]
## Hello Everyone Adil here 
I am new to GitHub can u give me a complete guide how should i get started with Github

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arunkarthik-periyaswamy** [comment]
Hello Adil, 

You can check this github's skills section to learn: https://github.com/skills/introduction-to-github

I hope this helps you.

**Comment by rituraj-gautam** [comment]
You can watch this video for better understamding - https://www.youtube.com/watch?v=Ez8F0nW6S-w&pp=ygUPZ2l0aHViIHR1dG9yaWFs

**Comment by ebndev** [comment]
Hi @adil-wq, thanks for being a part of the GitHub Community!

Our DevRel team put together a great series titled: [🎥 Let's learn git and GitHub together! ](https://github.com/orgs/community/discussions/126301) that can help you get started on the basics.

Best of luck on your journey 🌟

**Comment by romhenri** [comment]
Hi, man.

You can use Github as:

### GitHub as Code Cloud
- Centralized Storage: Store and manage code in the cloud.
- Version Control: Track changes and revert to previous versions.
- Collaboration: Team work easily.

### GitHub as Portfolio
- Showcase: Display projects publicly.
- Contribution History: Record of contributions.
- Documentation: Add project details with READMEs.

### GitHub for Open-Source Contributions
- Contribution: Submit your updates and issues to projects.
- Learning: Learn from and with other developers projects.
- Community: Collaborate with open-source project.

To get the most out of Github, it is essential to know git.
